src/CarbonAware.CLI/test/unitTests/Commands/EmissionsCommandTests.cs

```diff
@@ -0,0 +1,167 @@
+using CarbonAware.Aggregators.CarbonAware;
+using CarbonAware.CLI.Commands.Emissions;
+using CarbonAware.Model;
+using Microsoft.Extensions.Options;
+using Moq;
+using NUnit.Framework;
+using NUnit.Framework.Internal;
+using System.Text.Json;
+
+namespace CarbonAware.CLI.UnitTests;
+
+[TestFixture]
+public class EmissionsCommandTests : TestBase
+{
+    [Test]
+    public async Task Run_CallsAggregatorAndWritesResults()
+    {
+        // Arrange
+        var emissionsCommand = new EmissionsCommand();
+        var invocationContext = SetupInvocationContext(emissionsCommand, "emissions");
+        var expectedLocationJSON = "\"Location\":\"eastus\"";
+        var expectedTimeJSON = "\"Time\":\"2022-01-01T00:00:00+00:00\"";
+        var expectedDurationJSON = "\"Duration\":\"01:00:00\"";
+        var expectedRatingJSON = "\"Rating\":100.7";
+
+        var emissionsData = JsonSerializer.Deserialize<EmissionsData>(
+            "{" + $"{expectedLocationJSON},{expectedTimeJSON},{expectedDurationJSON},{expectedRatingJSON}" + "}")!;
+
+        _mockCarbonAwareAggregator.Setup(agg => agg.GetEmissionsDataAsync(It.IsAny<CarbonAwareParameters>()))
+            .ReturnsAsync(new List<EmissionsData>() { emissionsData });
+
+        // Act
+        await emissionsCommand.Run(invocationContext);
+
+        // Assert
+        StringAssert.Contains(expectedLocationJSON, _console.Out.ToString());
```

Comment:
Maybe store the ToString() somewhere. I think it would create everytime a string this way.